i18n-embed/src/fluent.rs

```diff
@@ -257,6 +257,19 @@ impl FluentLanguageLoader {
 
         (closure)(&mut iter)
     }
+
+    /// Set whether the underlying Fluent logic should insert Unicode
+    /// Directionality Isolation Marks around placeables.
+    ///
+    /// See [`fluent::bundle::FluentBundleBase::set_use_isolating`] for more
+    /// information.
+    ///
+    /// Default: `true`.
+    pub fn set_use_isolating(&self, value: bool) {
```

Comment:
perhaps we could add a unit test for this?

Reply:
Yes, perhaps it would sufficient to check the byte lengths of the output strings? I'll add a commit today.

Reply:
I guess so, or to be more specific you could check if the string contains the expected special characters or not. One test with it enabled, and one without it enabled.